cmd/scheduled-feed/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/ossf/package-feeds/feeds/scheduler"
+
+	log "github.com/sirupsen/logrus"
+
+	"gocloud.dev/pubsub"
+	_ "gocloud.dev/pubsub/gcppubsub"
+)
+
+var delta = 5 * time.Minute
+
+func Poll(w http.ResponseWriter, r *http.Request) {
+	topicURL := os.Getenv("OSSMALWARE_TOPIC_URL")
+	topic, err := pubsub.OpenTopic(context.TODO(), topicURL)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	cutoff := time.Now().UTC().Add(-delta)
+	pkgs, err := scheduler.PollScheduledFeeds(cutoff)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Logging errors here similar to other error conditions would be helpful and consistent.